### DISCUSSION TITLE
Page not found

### DISCUSSION THREAD

**Discussion by emdadul38** [kinngh/shopify-node-express-mongodb-app#57] [General]
During installing the apps in the store Test your app on a development store without writing the URL, showing the message, and how to solve the issue. 
SHOPIFY_APP_URL/?hmac=da7221e561f28f60b5b1b5b4cd27aac083adbf4a1c4dd12da016562c82e04bd8&host=YWRtaW4uc2hvcGlmeS5jb20vc3RvcmUvZW1kYWQtMzg&shop=myshop.myshopify.com&timestamp=1677061190

![Screenshot from 2023-02-22 16-14-46](https://user-images.githubusercontent.com/10665798/220590512-2b3b36d8-e066-4155-9593-ee963018cd4c.png)

**Comment by kinngh** [comment]
1. Stop opening discussions when this should be an issue.
2. Please read `SETUP.md` when you're setting up the repo that has detailed instructions on how to get the repo up and running.
